src/validators/tasks.py

```diff
@@ -229,12 +287,24 @@ async def create_approval_request(
         deadline=deadline,
     )
     for validator in validators:
-        shards = await keystore.get_exit_signature_shards(
-            validator_index=validator_index,
-            public_key=validator.public_key,
-            oracles=oracles,
-            fork=settings.network_config.SHAPELLA_FORK,
-        )
+        if keystore is None:
+            if validator.exit_signature is None:
+                raise RuntimeError('validator.exit_signature must be set')
+
+            shards = await LocalKeystore.get_exit_signature_shards_without_keystore(
```

Comment:
Lets generate shards the same way: for local keystore get the exit signature and shard it then

Reply:
Thats the way it works